lib/screens/stake_screen/stake_screen.dart

```diff
@@ -0,0 +1,340 @@
+import 'dart:ui';
+
+import 'package:deus/core/widgets/dark_button.dart';
+import 'package:deus/core/widgets/filled_gradient_selection_button.dart';
+import 'package:deus/core/widgets/selection_button.dart';
+import 'package:deus/core/widgets/svg.dart';
+import 'package:deus/statics/my_colors.dart';
+import 'package:deus/statics/styles.dart';
+import 'package:flutter/material.dart';
+
+enum StakesStates {
+  hasToApprove,
+  pendingApprove,
+  canStake,
+  stakePendingApprove,
+  stakeApproved
+}
+
+class StakeScreen extends StatefulWidget {
+  static const url = '/stake';
+
+  @override
+  _StakeScreenState createState() => _StakeScreenState();
+}
+
+class _StakeScreenState extends State<StakeScreen> {
+  final String address = '0x3a6dabD6B5C75291a3258C29B418f5805792a87e';
+
+  final double apy = 99.93;
+
+  final balance = 13.234;
+
+  final gradient = MyColors.blueToGreenGradient;
+
+  StakesStates _stakeState = StakesStates.hasToApprove;
+  bool _showToast = false;
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: Color(MyColors.Background),
+      body: SafeArea(
+        child: SingleChildScrollView(
+          padding: const EdgeInsets.symmetric(vertical: 20, horizontal: 15),
+          child: SizedBox(
+            height: MediaQuery.of(context).size.height-80,
+            child: Column(
+              mainAxisSize: MainAxisSize.min,
+              crossAxisAlignment: CrossAxisAlignment.start,
+              children: [
+                _buildHeaderAddress(),
+                SizedBox(
+                  height: 23,
+                ),
```

Comment:
- [ ] const davor für bessere Performance, muss dann nicht immer neu gerendert werden.
- [ ] Würde es in height: 20 umändern und dann alle SizedBoxes mit height: 20 in eine konstante Variable `kSpacer` abändern. Macht den Code auch lesbarer.
- [ ] ggf. auch const kSmallSpacer mit height 12 oder so

Reply:
Das ist nach wie vor offen.